### PR TITLE
[Dns] Resolve new ApiView comments

### PR DESCRIPTION
**Pull request by dvbb**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs [hunk 1]
```diff
@@ -112,7 +112,7 @@ public partial class DnsZoneResource : Azure.ResourceManager.ArmResource
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dns.RecordSetAaaaResource>> GetRecordSetAaaaAsync(string relativeRecordSetName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.ResourceManager.Dns.RecordSetAaaaCollection GetRecordSetAaaas() { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dns.RecordSetAResource>> GetRecordSetAAsync(string relativeRecordSetName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.ResourceManager.Dns.RecordSetACollection GetRecordSetAs() { throw null; }
+        public virtual Azure.ResourceManager.Dns.RecordSetACollection GetRecordSetACollections() { throw null; }
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:115]: GetRecordSetACollection

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs [hunk 2]
```diff
@@ -549,12 +549,12 @@ public partial class DnsMXRecord
...
     {
         public DnsNSRecord() { }
-        public string NsdName { get { throw null; } set { } }
+        public string DnsNsdName { get { throw null; } set { } }
     }
     public partial class DnsPtrRecord
     {
         public DnsPtrRecord() { }
-        public string PtrdName { get { throw null; } set { } }
+        public string DnsPtrdName { get { throw null; } set { } }
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:557]: Should be `Ptr`, what is `Ptrd`?

Comment (dvbb) [reply sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:557]: The name of the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01/dns.json#L1209) definition is prtdname

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:557]: Ok, according to the swagger, the name should be `PtrDomainName`

**Pull request merged**